src/components/AlbumsList.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Album } from "../modules/posts/types";
+import {Card, Col, Row} from "antd";
```

Comment:
Put space between imports and curly braces and use single quotes only here and everywhere:
```import { Card, Col, Row } from 'antd';```